R/standalone-input-names.R

```diff
@@ -0,0 +1,83 @@
+# ---
+# repo: tidymodels/workflows
```

Comment:
This "standalone" file is in the next version of workflows and is designed to be added to other packages. It is tested there and the `nocov` directives prevent it from decreasing code coverage here.